bgpd/bgp_evpn.c

```diff
@@ -1522,13 +1539,21 @@ int evpn_route_select_install(struct bgp *bgp, struct bgpevpn *vpn,
 			evpn_delete_old_local_route(bgp, vpn, dest,
 					old_select, new_select);
 	} else {
-		if (old_select && old_select->type == ZEBRA_ROUTE_BGP
-		    && old_select->sub_type == BGP_ROUTE_IMPORTED)
-			ret = evpn_zebra_uninstall(
-				bgp, vpn,
-				(const struct prefix_evpn *)bgp_dest_get_prefix(
-					dest),
-				old_select, false);
+		if (old_select && old_select->type == ZEBRA_ROUTE_BGP &&
+		    old_select->sub_type == BGP_ROUTE_IMPORTED) {
+			if ((CHECK_FLAG(bgp->flags,
+					BGP_FLAG_DELETE_IN_PROGRESS)) ||
+			    (CHECK_FLAG(bgp->flags, BGP_FLAG_VNI_DOWN)))
+				(void)evpn_zebra_uninstall(bgp, vpn,
```

Comment:
Why void?

Reply:
Just a coding habit to ignore the return value to satisfy SA in previous company :). Will remove this..